.github/workflows/release-binaries.yaml

```diff
@@ -48,9 +48,13 @@ jobs:
       run: |
         set -x
 
-        # Install the latest version of hub
+        # Install the latest version of hub.
         ./build/install-hub.sh
 
-        # Create the release with all binaries
+        # Get a commit log since the previous tag.
+        previous_tag="$(git tag | grep -v '{{ github.ref }}' | grep '^v.*' | sort -Vr | head -1)"
+        changes="Changes:\n$(git log --pretty='format:* %s (by %an) on %cD' ${previous_tag}..HEAD)"
+
+        # Create the release with all binaries.
         version="$(echo ${{ github.ref }} | sed -e 's|^refs/tags/||')"
-        bin/hub release create $(printf -- ' --attach=%s' ./*-amd64) -m "Binaries release version ${version}" "${version}"
+        bin/hub release create $(printf -- ' --attach=%s' ./*-amd64) -m "Binaries release version ${version}" -m "${changes}" "${version}"
```

Comment:
the `changes` variable includes all the changes from the previous tag including changes related to policygen and templates. So, probably `Binaries release version vx.x.x` isn't a good title anymore.

Reply:
Updated it to only consider paths with Go source files.

Reply:
Can you please update RELEASING.md and include a new bullet point mentioning that users should Edit/Update release notes as required? Some PRs are not important and including them in the notes adds clutter and can confuse users.

Reply:
Done.